src/main/java/seedu/address/model/person/DateRepaid.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.model.person;
+
+/**
+ * Represents the date of when the Person was sent to the whitelist in the address book, i.e. the date
+ * the Person repays his debt completely.
+ * Guarantees: immutable;
+ */
+public class DateRepaid {
+
+    public static final String STANDARDISED_PLACEHOLDER = "NOT REPAID";
```

Comment:
name is not developer-intuitive enough. consider "DATE_REPAID_PLACEHOLDER" or "MESSAGE_NOT_REPAID" or other names along those lines

Reply:
Noted. Thanks for the suggestion!